### PR TITLE
CRB-149: Enable existing tests and start collecting coverage

### PR DESCRIPTION
**Pull request by atodorov**
I've just realized this repo has both dev and main branches. Opening against dev.

@will-gluwa, @CAGS295 maybe you want to remove main here?

### CODE REVIEW

**Review by CAGS295 — APPROVED** [review]

**Comment by CAGS295** [comment]
> @will-gluwa, @CAGS295 maybe you want to remove main here?

What do you mean by 'removing main here'?

**Comment by atodorov** [comment]
> What do you mean by 'removing main here'?

Remove the branch called "main" and leave the branch called "dev" as the default one.

**Comment by atodorov** [comment]
@will-gluwa, @nathanwhit I need 1 more approval here before GitHub will let me click on the merge button.

**Pull request merged**
